go/stats/utils.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"database/sql"
+	"github.com/inverse-inc/packetfence/go/database"
+	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
+)
+
+// connectDB connect to the database
+func connectDB(configDatabase pfconfigdriver.PfConfDatabase, db *sql.DB) {
```

Comment:
use the database package instead

Reply:
Unclear, same response as Same response as https://github.com/inverse-inc/packetfence/pull/3024#discussion_r175586997